bin/storer.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3.6
+
+import argparse, signal, sys
+from concurrent.futures import ProcessPoolExecutor
+from urllib3.util import parse_url, Url
+from bundle_tools import logger, StagedBundle, StagedBundleFinder, BundleStorer, DataStoreAPI
+
+"""
+    Store staged bundles in the HCA DSS Data Store
+    
+    Bundle must have been previously staged.
+    s3 URL of a bundle or bundle hierarchy (which has at least one 'bundles' subfolder) must be supplied, e.g.
+    
+    bin/storer.py --bundles s3://org-humancellatlas-data-bundle-examples/import/10x
+    
+    bin/storer.py --bundle s3://org-humancellatlas-data-bundle-examples/import/geo/GSE75478/bundles/bundle145
+"""
+
+
+# Executor complains if it is an object attribute, so we make it global.
+executor = None
+
+
+class Main:
+
+    DEFAULT_STAGED_BUNDLES_URL = "s3://org-humancellatlas-data-bundle-examples/import"
+
+    def __init__(self):
+        parser = argparse.ArgumentParser(description="Store staged bundles in HCA Data Store")
+        parser.add_argument('--bundles', default=self.DEFAULT_STAGED_BUNDLES_URL,
+                            help="S3 URL of staged bundles")
+        parser.add_argument('--bundle',
+                            help="S3 URL of a single staged bundle")
+        parser.add_argument('--smaller', type=int, default=1024*1024,  # 1 TB
+                            help="only store bundles with data files smaller than this many MB")
+        parser.add_argument('--dss-endpoint', default=DataStoreAPI.DEFAULT_DSS_URL,
+                            help="URL of DSS API endpoint")
+        parser.add_argument('-r', '--use-rest-api', action='store_true', default=False,
+                            help="use REST API directly, instead of python bindings")
+        parser.add_argument('-j', '--jobs', type=int, default=1,
+                            help="parallelize with this many jobs (implies --terse)")
+        parser.add_argument('-q', '--quiet', action='store_true', default=False,
+                            help="silence is golden")
+        parser.add_argument('-t', '--terse', action='store_true', default=False,
+                            help="terse output, one character per file")
+        parser.add_argument('-i', '--report-task-ids', action='store_true', default=False,
+                            help="when ")
+        parser.add_argument('-l', '--log', default=None,
+                            help="log verbose output to this file")
+        self.args = parser.parse_args()
+        if self.args.jobs > 1:
+            quiet = True
+            terse = True
```

Comment:
You can assign to `self.args.quiet = True`, I don't think there's a good reason not to do that.